src/screens/GroupSetting/hooks/useGroupSetting.js

```diff
@@ -56,12 +56,13 @@ const useGroupSetting = ({route}) => {
   }, [selectedChannel]);
 
   const submitData = async (withNavigation = true) => {
-    if (!changeImage || changeName) return navigation.goBack();
+    if (!changeImage && !changeName) return navigation.goBack();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition for returning from the function has been changed from an OR to an AND operation. This means that if either `changeImage` or `changeName` is false, the function will not return and continue execution. If this is not the intended behavior, consider reverting it back.
```diff
-    if (!changeImage || changeName) return navigation.goBack();
+    if (!changeImage && !changeName) return navigation.goBack();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->